apps/lib/s_cb.c

```diff
@@ -746,6 +746,12 @@ static STRINT_PAIR tlsext_types[] = {
     {"psk kex modes", TLSEXT_TYPE_psk_kex_modes},
     {"certificate authorities", TLSEXT_TYPE_certificate_authorities},
     {"post handshake auth", TLSEXT_TYPE_post_handshake_auth},
+#ifndef OPENSSL_NO_ECH
+    {"encrypted ClientHello (draft-13)", TLSEXT_TYPE_ech},
+    {"outer exts", TLSEXT_TYPE_outer_extensions},
+    /* not an ECH change, but seems to be missing */
+    {"early_data", TLSEXT_TYPE_early_data},
```

Comment:
is early_data  dependent of ECH ?
Else the #endif should move up, no ?

Reply:
Not dependent, no. I just added that as it made reading logs easier when I was testing and kept it within the ifdef , but sure, moving it out is better so did that.